test/unit/init/variables/extractors/BaseUrlExtractor.test.ts

```diff
@@ -16,6 +16,10 @@ describe('A BaseUrlExtractor', (): void => {
     await expect(computer.handle({ port: 3333 })).resolves.toBe('http://localhost:3333/');
   });
 
+  it('uses Unix Domain Sockets if socket is set.', async(): Promise<void> => {
```

Comment:
Description doesn't match the test. Also check the string that is being thrown in the test itself to make sure the correct error is thrown.